src/components/list/StockList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { Card } from "antd";
+import { type tFakeDataStocks , mappingStockType} from "../../types/stock";
```

Comment:
Use forward import with aliase.
```suggestion
import { type tFakeDataStocks , mappingStockType} from "~/types/stock";
```